packages/server/tests/acceptance/estimateGasPrecompile.spec.ts

```diff
@@ -446,14 +709,14 @@ describe('EstimatePrecompileContract tests', function () {
     let account0LongZero = Utils.idToEvmAddress(account0Wallet.account);
     let account1LongZero = Utils.idToEvmAddress(account1Wallet.account);
 
-    const ERC20Contract = new ethers.Contract(tokenAddress, ERC20MockJson.abi, accounts[0].wallet);
+    //const ERC20Contract = new ethers.Contract(tokenAddress, ERC20MockJson.abi, accounts[0].wallet);
 
-    const erc20Accounts0 = ERC20Contract.connect(accounts[0].wallet);
+    const erc20Accounts0 = tokenContract.connect(accounts[0].wallet);
     // @ts-ignore
     const tx1 = await erc20Accounts0.approve(contract.target, 2);
     await tx1.wait();
 
-    const allowance = await ERC20Contract.allowance(accounts[0].wallet.address, contract.target);
+    const allowance = await tokenContract.allowance(accounts[0].wallet.address, contract.target);
     await new Promise((r) => setTimeout(r, 5000));
```

Comment:
Take a look if all of the Timeouts are needed.
(we spoke offline and you said they are not)

Reply:
@ebadiere is working on another PR to replace the timeout for a `polling` mechanism, maybe you can take a look at that util/helper. Is still a work in progress but the idea is to allow more time, and at the same time to get a response faster.

https://github.com/hashgraph/hedera-json-rpc-relay/pull/2031/files

Reply:
Removed the unneeded timeouts as they are not needed